docs/automated-workflows/dagman-introduction.rst

```diff
@@ -60,18 +60,18 @@ Example: Diamond DAG
          A --> B & C
          B & C --> D
 
-A simple diamond-shaped DAG, as shown in the following image is presented as a
+A simple diamond-shaped DAG, as shown in the image on the right is presented as a
 starting point for examples. This diamond DAG contains 4 nodes. This diamond-shaped
```

Comment:
Could we *be* any more wordy?  How about something like

A diamond-shaped DAG, as shown on the right, will be used in examples.